### PR TITLE
Updating the latest contracts version

### PR DESCRIPTION
**Pull request by aaitor**
## Description

Using contracts `v1.1.4`

## Is this PR related with an open issue?

https://github.com/nevermined-io/internal/issues/153

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
 
## Checklist:

- [X] Follows the code style of this project.
- [X] Tests Cover Changes
- [X] Documentation

### CODE REVIEW

File: .bumpversion.cfg [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 0.4.3
+current_version = 0.7.4
```
Comment (r-marques) [inline .bumpversion.cfg:2]: I think this should also be 0.7.6

Comment (r-marques) [reply .bumpversion.cfg:2]: Although I personally don't use the bumpversion

**Review by r-marques — APPROVED** [review]
lgtm

**Comment by aaitor** [comment]
blocked by https://github.com/nevermined-io/metadata-api/issues/22

**Pull request merged**